pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -51,6 +51,23 @@ func (s *ServiceImpl) addAppLinks(treeRoot *navtree.NavTreeRoot, c *contextmodel
 		}
 	}
 
+	// Update select tree nodes for certain plugins
+	for _, plugin := range s.pluginStore.Plugins(c.Req.Context(), plugins.TypeApp) {
```

Comment:
I'm curious about your thoughts on why here versus in the subfunction of the `for` loop above. Another product, SLO, is manually added to a place in the nav in that function here: https://github.com/grafana/grafana/blob/355b3fb32eac8eb2a5d575ce478a35909231eb9c/pkg/services/navtree/navtreeimpl/applinks.go#L187 so I'd think following that would be helpful for future refactoring.

I'm not sure which is best, honestly :)